### PR TITLE
Feature/fact execution

### PR DESCRIPTION
**Pull request by campisgabriel**
# Overview
Creation of fact_executions models. I still need to document this model. I also noticed that in Snowflake we can't get from the dbt logs the information about bytes processed. We probably need to study how to get this information, maybe in the snowflake information schema.

# key changes
stagings from elementary raw
a model in marts called fact_executions created

### CODE REVIEW

File: models/dbt_sf_tranform_monitoring/source/src_elementary.yml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            data_type: string
+            description: Start time of resource compile action.
+
+          - name: compile_completed_at
+            data_type: string
+            description: End time of resource compile action.
+
+          - name: rows_affected
+            data_type: int
+            description: ""
```
Comment (vitorweiss) [inline models/dbt_sf_tranform_monitoring/source/src_elementary.yml:201]: legal ja deixar uma description simples aqui

File: models/dbt_sf_tranform_monitoring/source/src_elementary.yml [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+            data_type: string
+            description: The unique id of the resource (would be similar for all executions of the same resource).
+
+          - name: invocation_id
+            data_type: string
+            description: The unique id of the invocation (would be similar for all resources executed on the same invocation). FK to dbt_invocations.
+
+          - name: generated_at
+            data_type: string
+            description: ""
```
Comment (vitorweiss) [inline models/dbt_sf_tranform_monitoring/source/src_elementary.yml:161]: aqui tb

File: models/dbt_sf_tranform_monitoring/source/src_elementary.yml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+          - name: thread_id
+            data_type: string
+            description: Id of the thread of this resource run.
+
+          - name: adapter_response
+            data_type: string
+            description: Response returned by the adapter (Fields will be different for each adapters).
+
+      - name: dbt_source_freshness_results
```
Comment (vitorweiss) [inline models/dbt_sf_tranform_monitoring/source/src_elementary.yml:227]: pq essas n tao documentadas?

Comment (campisgabriel) [reply models/dbt_sf_tranform_monitoring/source/src_elementary.yml:227]: because it's not modeled yet

**Review by vitorweiss — APPROVED** [review]

**Pull request merged**